src/api/code-challenge/content-types/code-challenge/lifecycles.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  beforeCreate(event) {
+    console.log("**** beforeCreate hook is activated! ****");
```

Comment:
The console.log can be used for testing but shouldn't be merged. Make sure to remove it when you finish. If anything we would do more specific logging , but this is too generic